client/src/components/ModalAddGlider.vue

```diff
@@ -117,6 +125,8 @@ onMounted(() => {
   modal.value = new Modal(_modal.value);
 });
 
+onUnmounted(() => modal.value?.hide());
```

Comment:
Warum rufst du das hier nochmal explizit auf?

Außerdem hast du in 131 auch explizit eine Hide-Funktion definiert. Nutze doch direkt diese.

Reply:
Sonst kann es passieren, dass der abgedunkelte Hintergrund vom Modal aktiv bleibt wenn man im Modal auf den Link klickt. Als best practice sollten wir das generell noch überall einbauen, wo ein Modal verwendet wird.